upgrade/src/pages/api/og.tsx

```diff
@@ -0,0 +1,75 @@
+import { ImageResponse } from "@vercel/og";
```

Comment:
ImageResponse is bundled in next/server

No need to install @vercel/og explicitely